core/src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -105,7 +112,7 @@ public InternalClusterInfoService(Settings settings, NodeSettingsService nodeSet
         public void onRefreshSettings(Settings settings) {
             TimeValue newUpdateFrequency = settings.getAsTime(INTERNAL_CLUSTER_INFO_UPDATE_INTERVAL, null);
             // ClusterInfoService is only enabled if the DiskThresholdDecider is enabled
```

Comment:
This comment is now misleading and the import for DiskThresholdDecider is now unused.  By the way, what will happen if we disable cluster info but will not disable disk threshold decider? Should we add a test for it, to make sure that it still produces sane results?